README.adoc

```diff
@@ -30,6 +30,7 @@ breaker implementation that best fits your needs for your app.
 
 * https://github.com/Netflix/Hystrix[Netfix Hystrix]
 * https://github.com/resilience4j/resilience4j[Resilience4J]
+* https://github.com/alibaba/Sentinel[Sentinel]
```

Comment:
No need to make changes to this file, it is automatically generated, see the warning at the top of the file

Reply:
Thanks for reminding. I've recovered the changes.